nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -135,6 +135,13 @@ let
   };
 
 in {
+  options.boot.initrd.environment.etc = mkOption {
+    visible = false;
+    default = {};
+    type = options.environment.etc.type;
```

Comment:
Before merging: How should we share this type, if at all? This way does not work when building the nixos manual, though it was awfully convenient for developing the feature.